Index: src/TestSB/TestSB_201.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package TestSB;\r\n\r\npublic class TestSB_201 {\r\n    //conflict practise\r\n\r\n}\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/TestSB/TestSB_201.java b/src/TestSB/TestSB_201.java
--- a/src/TestSB/TestSB_201.java	(revision 23c8d8ce75d6115d065108260822977b8e092b2e)
+++ b/src/TestSB/TestSB_201.java	(date 1713852295799)
@@ -1,6 +1,6 @@
 package TestSB;
 
 public class TestSB_201 {
-    //conflict practise
+    //conflict practise crated
 
 }
